### DISCUSSION TITLE
I have issues with installing Distro AV

### DISCUSSION THREAD

**Discussion by JeffJrShim** [DistroAV/DistroAV#1133] [Q&A - Support] [answered]
![image](https://github.com/user-attachments/assets/fbb208fd-98f3-4b70-ac0f-2de7b4367627)
This is what I got. I already uninstalled OBS-NDI, but nothing seemed to work. I've tried many ways to see how to fix it and sadly i am unable to find a fix to this issue,

**Comment by BitRate27** [comment, marked as answer]
Sorry for the troubles. Looks like you are running Windows. If you do not Close OBS, before you uninstall obs-ndi, the .dll will not be removed. You may have received a message saying not all components were removed, but obs-ndi will no longer show in the installed programs list. If this has happened you must remove C:\Program Files\obs-studio\obs-plugins\64bit\obs-ndi.dll manually.

**Reply by JeffJrShim** [reply]
Thank you very much!

**Reply by GWStill** [reply]
Hi,
I'm having the same issue on my MAC. I found the folder in application support but the OBS NDI plugin in not there. I uninstalled this plugin as well as the NDI tools, and reinstalled this file. The error message is still there. I don't know what I'm missing. Please help. Thank you in advance.

**Reply by BitRate27** [reply]
It may be in the system library /Library/Application Support... (note the missing ~).

**Comment by GWStill** [comment]
This is what I see.


Deacon Sharon Tillman

Media Administrator

[A picture containing logo  Description automatically generated]

642-44 Crotona Pk. So.

Bronx, New York 10456

718.618.6700 | goodwillbc.org

________________________________

**Reply by GWStill** [reply]
![Image 12-23-24 at 2 18 PM](https://github.com/user-attachments/assets/a3a6193e-abbd-4cba-98c8-85340aa1e43d)

**Reply by BitRate27** [reply]
That looks like it may be the user folder.

To find the system folder, bring up Finder and do Shift-Command-G. Then paste this location: /Library/Application Support/obs-studio/plugins

If you see a obs-ndi.plugin there, delete it. This could happen if the person installing the obs-ndi plugin installed it for all users or installed it as admin.

**Comment by GWStill** [comment]
Thank you for your help! I really appreciate it ☺️

Deacon Sharon Tillman
Media Administrator
[A picture containing logo  Description automatically generated]
642-44 Crotona Pk. So.
Bronx, New York 10456
718.618.6700 | goodwillbc.org

________________________________
